### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by Mathieu4141**
Tested it on testpypi (v0.23.1.1), the IoC example runs successfully with this setup config

### CODE REVIEW

**Review by LouisTrezzini — COMMENTED** [review]
I don't think the want the examples to be installed with the library. Shouldn't they live at the root? `/examples`, next to `/feedly`

File: setup.py [hunk 1]
```diff
@@ -102,12 +108,7 @@ def run(self):
     author_email=EMAIL,
     python_requires=REQUIRES_PYTHON,
     url=URL,
-    packages=["feedly.api_client"],
```
Comment (LouisTrezzini) [inline setup.py:105]: can we keep the previous value? (explicit is better than implicit)

File: feedly/api_client/utils.py [hunk 2]
```diff
@@ -3,9 +3,8 @@
 from typing import Callable
```
Comment (LouisTrezzini) [inline feedly/api_client/utils.py:3]: can this file live in the examples folder? it's not useful in the library

**Review by LouisTrezzini — APPROVED** [review]

File: feedly/api_client/utils.py [hunk 3]
```diff
@@ -3,9 +3,8 @@
...
 
 from feedly.api_client.protocol import UnauthorizedAPIError
-from feedly.examples.setup_auth import setup_auth
 
-EXAMPLES_DIR = Path(__file__).parent
+EXAMPLES_DIR = Path(__file__).parent.parent.parent / "examples"
```
Comment (LouisTrezzini) [inline feedly/api_client/utils.py:7]: this is inaccurate (eg if the library is installed in a venv)

File: feedly/api_client/utils.py (hunk 2 above)
Comment (Mathieu4141) [reply feedly/api_client/utils.py:3]: imports won't work if it's in the examples folder since we moved it out of the lib

**Pull request merged**